README.md

```diff
@@ -4,7 +4,7 @@ Palestine flag banner to show support from your website.
 
 ### Installation
 ```
-<script src="https://cdn.jsdelivr.net/gh/devkayes/we-stand-with-palestine@main/ribbon-banner.js" async></script>
+<script src="https://cdn.jsdelivr.net/gh/samiulhafiz/we-stand-with-palestine@main/ribbon-banner.js" async></script>
```

Comment:
While you merge your main branch into the source branch, you need to provide the source branch repository link, not your source branch link. Your code is completely fine, but when you request the PR, your `README.md` changes will overwrite mine. 

In that case, either you have to change the `README.md` content or make a change to the content in another branch and then submit the PR with the newly created branch again (best practice).  

The updated README.md content should be: `<script src="https://cdn.jsdelivr.net/gh/devkayes/we-stand-with-palestine@main/ribbon-banner.js" async></script>`

Let me know if you need help with that; thanks.